snowpack/src/commands/build.ts

```diff
@@ -105,8 +105,9 @@ export async function build(commandOptions: CommandOptions): Promise<SnowpackBui
     const files = (await new fdir().withFullPaths().crawl(mountKey).withPromise()) as string[];
     const excludePrivate = new RegExp(`\\${path.sep}\\.`);
     const excludeGlobs = [...config.exclude, ...config.testOptions.files];
+    const foundExcludeMatch = picomatch(excludeGlobs);
     for (const f of files) {
-      if (micromatch.isMatch(f, excludeGlobs) || excludePrivate.test(f)) {
+      if (picomatch(f) || excludePrivate.test(f)) {
```

Comment:
There's probably more perf gains to be had here by leveraging the `exclude` option in `fdir` like I'm doing in `dev.ts`. 

I don't have a  repro case for the build command in a large repo so I've made the most trivial change to replace `micromatch` with `picomatch`